definitions/tools/star_align_fusion.cwl

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env cwl-runner
+class: CommandLineTool
+cwlVersion: v1.0
+label: "STAR: align reads to transcriptome"
+baseCommand: ["/usr/local/bin/STAR"]
+requirements:
+    - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      ramMin: 64000
+      coresMin: 10
+    - class: DockerRequirement
+      dockerPull: "mgibio/star"
```

Comment:
could you add a version tag for the docker image?

Reply:
this did not work as expected although i added a release tag to the github repo, docker hub failed to add tag while automated build, have reached out to @zlskidmore to see if i missed anything.

Reply:
we have the docker hub integration to work on tags or branches. I created a branch `2.7.0f`  from the master branch on the docker-star repo. This triggered a build on docker hub. Once that finishes you can add the tag to the cwl

Reply:
thanks have added the tag as required.